features/git-kill/current_branch/on_feature_branch/without_tracking_branch/without_remote_origin.feature

```diff
@@ -0,0 +1,45 @@
+Feature: git kill: killing the current feature branch without a tracking branch (without open changes or remote origin)
```

Comment:
I don't think this should be nested under `without_tracking_branch`. Tracking branches aren't possible without a remote.

I'd say just move this test to its own folder `without_remote` which is a sibling folder of `without_tracking_branch` and please add a test for with open changes as well.